lib/modules/datasource/rubygems/get.ts

```diff
@@ -182,4 +200,87 @@ export class InternalRubyGemsDatasource extends Datasource {
 
     return new Marshal(response.body).parsed as T;
   }
+
+  private async isNexusDataSource(registry: string): Promise<boolean> {
+    const statusEndPoint = '/service/rest/v1/status';
+    const endPointRe = regEx(`^(?<endPoint>.*:\\d{1,5})\\/`);
+    const { endPoint } = registry.match(endPointRe)?.groups ?? {
+      endPoint: '',
+    };
+    const nexusUrl = endPoint.concat(statusEndPoint);
```

Comment:
So if `endpoint` falls back to `''` then you concat `statusEndPoint` to it, you then try to get that URL even though it has no hostname?

Reply:
I think this is same comment as made already by @PhilipAbed